src/tests/system/tests/test_authentication.py

```diff
@@ -48,6 +48,144 @@ def test_authentication__with_default_settings(
     ), "User logged in with an invalid password!"
 
 
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.importance("critical")
+def test_authentication__change_password(
+    client: Client,
+    provider: GenericProvider,
+):
+    """
+    :title: User logins and issues a password change
```

Comment:
The wording for the title seems difficult to read to me.   Maybe something like `user logs in and issues a password change`?  Or maybe just `user authenticates and issues a password change`?  Or `User issues a password change after login`?   I'm not sure which makes the most sense.

Reply:
I like the last one, it clear on what it does.

Reply:
Sounds good to me.